### PR TITLE
Adding docs for policy APIs and chef-zero

### PR DESCRIPTION
**Pull request by vinay-satish**
### Description

Updating the documentation of policy APIs and chef-zero

### Check List

- [ ] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/main/CONTRIBUTING.md#developer-certification-of-origin-dco>
- [ ] PR title is a worthy inclusion in the CHANGELOG

### CODE REVIEW

File: docs-chef-io/content/server/api_chef_server.md [hunk 1]
```diff
@@ -2432,9 +2271,7 @@ The response is similar to:
 
 #### POST
 
-The `POST` method is used to add a key for the specified client.
-
-This method has no parameters.
+The `POST` method is used to add a key for the specified client. This method has no parameters.
```
Comment (IanMadd) [inline docs-chef-io/content/server/api_chef_server.md:2274]: We should keep all the text saying "This method has no parameters." on a separate line.

File: dev-docs/HOW_THINGS_WORK.md [hunk 2]
```diff
@@ -194,126 +144,98 @@ object types and organizations.
...
+- The add and update triggers in keys\_update\_trigger.sql are present to maintain compatibility with older versions of chef-server.
+
+### Phase2(To be implemented)
+
+#### Todo
+
+- Delete and update triggers in keys\_update\_trigger.sql.
+- Delete the public\_key column in the users (and perhaps clients) table.
+
+{{< note >}}
```
Comment (tas50) [inline dev-docs/HOW_THINGS_WORK.md:213]: This isn't a page that gets rendered so this note syntax won't work

**Review by PrajaktaPurohit — APPROVED** [review]

File: dev-docs/HOW_THINGS_WORK.md (hunk 2 above)
Comment (kagarmoe) [inline dev-docs/HOW_THINGS_WORK.md:213]:
```suggestion
```

File: dev-docs/HOW_THINGS_WORK.md [hunk 3]
```diff
@@ -194,126 +144,98 @@ object types and organizations.
...
+
 - This should be easier once all the users are on Phase1 release of chef-server.
-- The code will then only interact with the add\_user and update\_user functions in
-the back.
-- The upgrade path after Phase2 rollout will include upgrading to a release with
-Phase1 first.
+- The code then interacts with the add\_user and update\_user functions in the back.
+- The upgrade path after Phase2 rollout includes upgrading to a release with Phase1 first.
+
+{{< /note >}}
```
Comment (kagarmoe) [inline dev-docs/HOW_THINGS_WORK.md:219]:
```suggestion
```

File: dev-docs/HOW_THINGS_WORK.md [hunk 4]
```diff
@@ -194,126 +144,98 @@ object types and organizations.
...
+
+### Phase2(To be implemented)
+
+#### Todo
+
+- Delete and update triggers in keys\_update\_trigger.sql.
+- Delete the public\_key column in the users (and perhaps clients) table.
+
+{{< note >}}
+
```
Comment (kagarmoe) [inline dev-docs/HOW_THINGS_WORK.md:214]:
```suggestion
Note:
```

File: docs-chef-io/content/server/api_chef_server.md (hunk 1 above)
Comment (kagarmoe) [inline docs-chef-io/content/server/api_chef_server.md:2274]:
```suggestion
The `POST` method is used to add a key for the specified client. 

This method has no parameters.
```

**Review by kagarmoe — CHANGES_REQUESTED** [review]

**Review by kagarmoe — APPROVED** [review]

**Pull request merged**